spec/support/views.rb

```diff
@@ -0,0 +1,15 @@
+module ViewStubHelpers
+  def signed_in=(signed_in)
+    @signed_in = signed_in
+  end
+
+  def signed_in?
+    @signed_in
+  end
+end
+
+RSpec.configure do |config|
+  config.before type: :view do
+    view.extend(ViewStubHelpers)
+  end
```

Comment:
@mrgilman @gabebw @joshuaclayton this seems to work without feeling too hacky.

What do you think?